rules-xml/src/main/java/org/jboss/windup/rules/apps/xml/DiscoverXmlFilesRuleProvider.java

```diff
@@ -49,8 +49,25 @@
     public Configuration getConfiguration(GraphContext context)
     {
         return ConfigurationBuilder.begin()
-                    
                     .addRule(FileMapping.from(".*\\.xml$").to(XmlFileModel.class))
+                    .addRule(FileMapping.from(".*\\.tld$").to(XmlFileModel.class))
```

Comment:
Good! But this should be implemented as a file-mappings.windup.xml rule file that gets included in our distribution. Please let me know if you have questions about how to do that!